core/init.js

```diff
@@ -0,0 +1,33 @@
+/*
+   init.js is appending to our minified bundles in the CP build process.
```

Comment:
I didn't see any reference to this being included anywhere (for fireplace at least). Shouldn't this be handled by setting the main module in requirejs anyway?